fileio/private/read_biosemi_bdf.m

```diff
@@ -60,8 +60,9 @@
   cname=computer;
   if cname(1:2)=='PC' SLASH=BSLASH; end;
 
-  fid=fopen(FILENAME,'r','ieee-le');
-  if fid<0
+  try
+    fid=fopen_or_error(FILENAME,'r','ieee-le');
+  catch err
     fprintf(2,['Error LOADEDF: File ' FILENAME ' not found\n']);
     return;
```

Comment:
I don't see why it would return here. This would only cause an error elsewhere (since the return value is not defined). I propose to remove the try-catch.